packages/manager/apps/telecom/src/app/telecom/telephony/service/contact/contact.constants.js

```diff
@@ -9,38 +9,180 @@ export const DIRECTORY_INFO = {
   },
 };
 
-export const DIRECTORY_WAY_NUMBER_EXTRA_ENUM = [
-  'bis',
-  'ter',
-  'quater',
-  'quinquies',
-  'sexto',
-  'septimo',
-  'octimo',
-  'nono',
-  'A',
-  'D',
-  'E',
-  'F',
-  'G',
-  'H',
-  'I',
-  'J',
-  'K',
-  'L',
-  'M',
-  'N',
-  'O',
-  'S',
-  'U',
-  'V',
-  'W',
-  'X',
-  'Y',
-  'Z',
-];
+export const DIRECTORY_WAY_NUMBER_EXTRA_ENUM = {
+  BE: [
+    '',
+    'A',
+    'D',
+    'E',
+    'F',
+    'G',
+    'H',
+    'I',
+    'J',
+    'K',
+    'L',
+    'M',
+    'N',
+    'O',
+    'S',
+    'U',
+    'V',
+    'W',
+    'X',
+    'Y',
+    'Z',
+  ],
+  OTHER: [
+    '',
+    'bis',
+    'ter',
+    'quater',
+    'quinquies',
+    'sexto',
+    'septimo',
+    'octimo',
+    'nono',
+    'A',
+    'D',
+    'E',
+    'F',
+    'G',
+    'H',
+    'I',
+    'J',
+    'K',
+    'L',
+    'M',
+    'N',
+    'O',
+    'S',
+    'U',
+    'V',
+    'W',
+    'X',
+    'Y',
+    'Z',
```

Comment:
I think you can do this :

```suggestion
const WAY_NUMBER_EXTRA = ['','A','D','E','F','G','H','I','J', 'K','L','M','N','O','S','U','V','W', 'X','Y','Z',];
export const DIRECTORY_WAY_NUMBER_EXTRA_ENUM = {BE: WAY_NUMBER_EXTRA.slice(), OTHER: ['bis',
    'ter',
    'quater',
    'quinquies',
    'sexto',
    'septimo',
    'octimo',
    'nono'].concat(WAY_NUMBER_EXTRA)};
```